VAPI-922 Update callback docs to be more specific (#315)
Bandwidth/api-docs: site/docs/voice/quickStart.mdx

## Configuring Callback URLs
The voice API can be configured to use multiple different callback URLs to direct traffic and receive input from phone calls.
Every application requires a `CallInitiatedCallbackUrl`, as this is the URL Bandwidth sends an [initiated webhook](./webhooks/initiate.mdx) to when an inbound call to a Bandwidth number is made,
and a BXML response is expected to continue the call.

> For inbound calls, Bandwidth will also send [disconnect webhooks](./webhooks/disconnect.mdx) to the configured `CallStatusCallbackUrl`. This webhook is useful for error checking and include the cause such as _rejected_ and _hung up_.

Reviewer: ```suggestion
For inbound calls, Bandwidth will also send [disconnect webhooks](./webhooks/disconnect.mdx) to the configured `CallStatusCallbackUrl`. This webhook is useful for error checking and includes the cause of the disconnect, for example _rejected_ or _hung up_.
```

These URLs can be the same or different, depending on if you would like inbound and outbound calls to be handled differently.

> ### Disconnect Callbacks

Reviewer 2: Should we have a paragraph for initiate?

Author: We kind of do above this but I could be convinced to roll both of these into a single paragraph

Reviewer 2: Yeah a single paragraph would be fine, imo
